backend/btrixcloud/orgs.py

```diff
@@ -276,6 +276,13 @@ async def get_org_for_user_by_id(
 
         return Organization.from_dict(res)
 
+    async def get_users_for_org(self, org: Organization, min_role=UserRole.VIEWER):
+        """get users for org"""
+        uuid_ids = [UUID(id_) for id_, role in org.users.items() if role >= min_role]
+        cursor = self.users_db.find({"id": {"$in": uuid_ids}})
+        user_datas = await cursor.to_list(length=1000)
+        return [User(**user_data) for user_data in user_datas]
```

Comment:
```suggestion
        users: List[User] = []
        async for user_dict in self.users_db.find({"id": {"$in": uuid_ids}}):
            users.append(User(**user_dict))
        return users
```

Slightly more efficient to iterate over the cursor async to avoid having to iterate through the list of results twice.